barman/output.py

```diff
@@ -792,30 +819,79 @@ def render_show_backup_base(backup_info, output_fun, header_row, nested_row):
             formatted
         """
         output_fun(header_row.format("Base backup information"))
-        if backup_info["size"] is not None:
-            disk_usage_output = "{}".format(pretty_size(backup_info["size"]))
-            if "wal_size" in backup_info and backup_info["wal_size"] is not None:
-                disk_usage_output += " ({} with WALs)".format(
-                    pretty_size(backup_info["size"] + backup_info["wal_size"]),
+        if backup_info["mode"] is not None:
+            output_fun(nested_row.format("Backup Method", backup_info["mode"]))
+        # Information only for postgres backups
+        if backup_info["mode"] == "postgres" and "backup_type" in backup_info:
+            output_fun(nested_row.format("Backup Type", backup_info["backup_type"]))
+            if backup_info["backup_type"] == "incremental":
+                output_fun(
+                    nested_row.format("Root Backup Id", backup_info["root_backup_id"])
                 )
-            output_fun(nested_row.format("Disk usage", disk_usage_output))
-        if backup_info["deduplicated_size"] is not None and backup_info["size"] > 0:
-            deduplication_ratio = 1 - (
-                float(backup_info["deduplicated_size"]) / backup_info["size"]
-            )
-            dedupe_output = "{} (-{})".format(
-                pretty_size(backup_info["deduplicated_size"]),
-                "{percent:.2%}".format(percent=deduplication_ratio),
-            )
-            output_fun(nested_row.format("Incremental size", dedupe_output))
+                output_fun(
+                    nested_row.format(
+                        "Parent Backup Id", backup_info["parent_backup_id"]
+                    )
+                )
+                output_fun(
+                    nested_row.format("Backup chain size", backup_info["chain_size"])
+                )
+            if backup_info["children_backup_ids"] is not None:
+                output_fun(
+                    nested_row.format(
+                        "Children Backup Id(s)",
+                        backup_info["children_backup_ids"],
```

Comment:
I think we should output this with something like `",".join(...)`.
The output will look better than Python representation of the list as a string.